simulator/simulator.py

```diff
@@ -0,0 +1,63 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+import random
+import datetime
+import time
+import json
+from pprint import pprint
+
+KEY_PATH = './firebase.json'
+DATABASE_URL = 'https://smartspace-84906-default-rtdb.firebaseio.com/'
+
+cred = credentials.Certificate(KEY_PATH)
+firebase_admin.initialize_app(cred, {
+        'databaseURL': DATABASE_URL
+})
+
+with open('coordinates.json') as json_file:
```

Comment:
```suggestion
with open(COORDINATES_CREDENTIAL_PATH) as json_file:
```

Suggest extract string constant to be a variable.

Reply:
The variable name is not exactly same, addressed at 59adfc0f54e8ccd5fd2f592b9a569dcafa7af726.